jenkins/e2e-image/Dockerfile

```diff
@@ -52,4 +52,7 @@ ADD ["e2e-runner.sh", \
     "/workspace/"]
 RUN ["chmod", "+x", "/workspace/get-kube.sh", "/workspace/upload-to-gcs.sh", "/workspace/sh2ju.sh"]
 
+ADD ["configure-image.sh", "/tmp"]
+RUN ["/tmp/configure-image.sh"]
```

Comment:
I think I didn't understand what the kubernetes-anywhere configuration was at the time.

I'd prefer keeping everything contained in the Dockerfile. It looks like basically 3 `RUN` lines and one `ENV` line.